.circleci/verbatim-sources/job-specs/pytorch-job-specs.yml

```diff
@@ -206,8 +206,10 @@ jobs:
               echo "Retrieving coverage report"
               docker cp $id:/var/lib/jenkins/workspace/test/.coverage ./test
               docker cp $id:/var/lib/jenkins/workspace/test/coverage.xml ./test
+              docker cp $id:/var/lib/jenkins/workspace/build/coverage.info ./test
               python3 -mpip install codecov
               python3 -mcodecov
+              python3 -mcodecov -f coverage.info
```

Comment:
This will upload the report twice, wouldn't it?

Reply:
Yes, I think it will. Fixed